java/ql/src/Security/CWE/CWE-094/InsecureBeanValidation.ql

```diff
@@ -14,6 +14,38 @@ import semmle.code.java.dataflow.TaintTracking
 import semmle.code.java.dataflow.FlowSources
 import DataFlow::PathGraph
 
+class ELMessageInterpolatorType extends RefType {
+  ELMessageInterpolatorType() {
+    this
+        .getASourceSupertype*()
+        .hasQualifiedName("org.hibernate.validator.messageinterpolation",
+          ["ResourceBundleMessageInterpolator", "ValueFormatterMessageInterpolator"])
+  }
+}
+
+class SetSafeMessageInterpolator extends MethodAccess {
```

Comment:
Add a QLDoc comment in `/** */` to explain. Something like: "A method call that sets the application's default message interpolator to an interpolator type that is likely to be safe, because it does not process Java Expression Language expressions."